content/docs/started/getting-started-k8s.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Kubeflow on Kubernetes"
+title = "Kubeflow on Local Kubernetes"
```

Comment:
s/Local/On-prem

This page is currently our solution for on-premises installations, although the process currently documented can also be used for local installations. We have a PR under review that adds LDAP auth to the page, and we want to leave the option open to provide on-prem alternatives to the Arrikto solution at some point in the future.

We should probably explain at the top of the page that these instructions are intended primarily for on-premises installations, although the process currently documented can also be used for local installations.

Reply:
Consider dropping this file from the commit? It is unrelated to the main focus of docker images.